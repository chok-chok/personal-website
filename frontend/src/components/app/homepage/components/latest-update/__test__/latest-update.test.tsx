import * as React from 'react';
import LatestUpdate from '../latest-update';
import { shallow } from 'enzyme';

describe('Test LatestUpdate component', () => {
  it('should renders correctly', () => {
    const tree = shallow(<LatestUpdate />);
    expect(tree).toMatchSnapshot();
  });
});
