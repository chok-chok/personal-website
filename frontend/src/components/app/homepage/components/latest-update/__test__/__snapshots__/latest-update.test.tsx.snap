// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test LatestUpdate component should renders correctly 1`] = `
<Fragment>
  <CurrentlyReading />
</Fragment>
`;
